autoscale-api/src/main/resources/messages/messages.properties

```diff
@@ -10,7 +10,7 @@ autoscale.enabled=Autoscaling has been enabled.
 autoscale.schedule.config.overlap=''SCHEDULE-BASED'' Autoscaling Configuration ''{0}'' not triggered, overlaps with ''{1}''.
 
 autoscale.cluster.exists.for.crn=Cluster exists for the same CRN ''{0}'' and CM Variant ''{1}''.
-autoscale.cluster.not.available=''{0}'' Autoscaling could not be triggered; cluster was not in 'AVAILABLE' status.
+autoscale.cluster.scaling.failed=''{0}'' Autoscaling could not be triggered. {1}
 autoscale.cluster.update.in.progress=''{0}'' Autoscaling could not be triggered; cluster update was in progress.
```

Comment:
Propogate CB API Message which has more context to user for reason of autoscale trigger failure like.
_SCHEDULE-BASED' Autoscaling could not be triggered. Stack smani-as has flows under operation, request not allowed._
_'SCHEDULE-BASED' Autoscaling could not be triggered. Upscale is not allowed because the following instances are not in running state: i-095969c15d35ee472. Please remove them first!._